addons/casings/CfgVehicles.hpp

```diff
@@ -0,0 +1,62 @@
+class CfgVehicles {
+    class NonStrategic;
+
+    class GVAR(FxCartridge): NonStrategic {
+        model = "\A3\Weapons_f\ammo\cartridge";
+        icon = "";
+        displayName = "";
+        nameSound = "";
+        destrType = "DestructNo";
+        mapSize = 0;
+        accuracy = 0;
+        armor = 5;
+        scope = 1;
+        typicalCargo[] = {};
+        transportAmmo = 0;
+        transportRepair = 0;
+        transportFuel = 0;
+        htMin = 30;
+        htMax = 60;
+        afMax = 400;
+        mfMax = 200;
+        mFact = 0;
+        tBody = 200;
+    };
+
+    class GVAR(FxCartridge_Small): GVAR(FxCartridge) {
+        model = QUOTE(PATHTOF(models\ace_casing_9.p3d));
+    };
+    class GVAR(FxCartridge_9mm): GVAR(FxCartridge_Small) {
+    };
+    class GVAR(FxCartridge_556): GVAR(FxCartridge) {
+        model = QPATHTO_M(models\ace_casing_556.p3d);
+    };
+    class GVAR(FxCartridge_65): GVAR(FxCartridge) {
+        model = QPATHTO_M(models\ace_casing_556.p3d);
+    };
+    class GVAR(FxCartridge_65_caseless): GVAR(FxCartridge) {
+        model = "\A3\weapons_f\empty";
```

Comment:
does it make sense to create invisible casings? shouldn't there be a better option for this, e.g. not creating them at all.